zilliqa/src/p2p_node.rs

```diff
@@ -112,6 +114,17 @@ impl P2pNode {
             )
             .map_err(|e| anyhow!(e))?,
             mdns: mdns::Behaviour::new(Default::default(), peer_id)?,
+            autonat: autonat::Behaviour::new(
+                peer_id,
+                autonat::Config {
```

Comment:
Is there a good reason we don't just use the default? If there is, can we add the reason(s) in a comment?